index.html

```diff
@@ -59,12 +60,16 @@ <h1 class="narson">I'm Narson.
                 </a>
             </li>
         </ul>
+    </div>
 
     </section>
 
     <section class="work">
         <div class="card-container">
-            <img class="sary1" src="media/Snapshoot Portfolio2.svg" alt="Project Snapshoot" />
+            <img class="sary" src="media/Snapshoot-Portfolio-desktop.svg" alt="Project Snapshoot" />
+            
+            <div class="work-fonony">
+
             <h2 class="card-tittle">Tonic</h2>
             <div class="frame2-img">
                 <img src="media/Frame 2.svg" />
```

Comment:
- [ ] Please make sure the short info about a project below the project heading is actual text content and not an image as this will ensure search engines and other assistive technologies can read it which helps the ranking of your website.
    -  Hint: you can use `ul` tag and wrap each piece of info in a `li` tag
 - This comment applies to all the info sections below the heading of each project.
     - Figma design:
       ![image](https://user-images.githubusercontent.com/104968236/230617793-ad30a91c-4522-4870-bc31-a669873e9c92.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230617882-16251ae6-2372-477f-975c-f31048c88090.png)